easybuild/easyconfigs/b/bcl-convert/bcl-convert-4.0.3.eb

```diff
@@ -0,0 +1,21 @@
+easyblock = 'Rpm'
+
+name = 'bcl-convert'
+version = '4.0.3'
+homepage = 'https://www.illumina.com/products/by-type/informatics-products/basespace-sequence-hub/apps/bcl-convert.html'
+description = """The BCL Convert App generates demultiplexed FASTQ files from a run as input. The core algorithm is BCL Convert.
+
+for more information: https://www.illumina.com/products/by-type/informatics-products/basespace-sequence-hub/apps/bcl-convert.html
+"""
+toolchain = {'name': 'system', 'version': ''}
+
+sources = ["https://webdata.illumina.com/downloads/software/%(name)s/%(name)s-%(version)s.el7.x86_64.rpm"]
```

Comment:
I could not download the RPM, is the download restricted? If so, please add a comment that registration/license is needed

Reply:
@lexming I didn't see any problem downloading the RPM. I did, however, need to install `rpmrebuild` on the build node.

Reply:
@verdurin https://webdata.illumina.com/downloads/software/bcl-convert/bcl-convert-4.0.3.el7.x86_64.rpm is throws a 404 for me. On the other hand, `rpmbuild` should not be needed to install any rpm package.